schemas/2015-05-21-preview/Microsoft.DevTestLab.json

```diff
@@ -21,13 +21,13 @@
       },
       "required": [ "type", "apiVersion", "location" ]
     },
-    "environments": {
-      "description": "Required properties to create environment in an existing DevTest Lab",
+    "virtualMachines": {
```

Comment:
When will your service start to support this resource type? This change will cause existing template be invalid.